### PR TITLE
Introduce the pull-request template.

### PR DESCRIPTION
**Pull request by oleg-nenashev**
This change explicitly sets expectations from the pull requests to simplify the maintenance.

Ideas:
* Highlight the need in autotests and JIRA issues for bugfixes
* Ask PR creators to propose the changelog entries for their changes.
* Provide hints about referencing people

CC @daniel-beck and @olivergondza

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Description
+
+See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).
+
+Details: TODO
+
+<!-- Comment:
```
Comment (oleg-nenashev) [inline PULL_REQUEST_TEMPLATE.md]: Comments are invisible in UI once the change is saved

**Comment by daniel-beck** [comment]
Should be in `.github`.

**Review by daniel-beck — APPROVED** [review]
Solid start, details can always be tweaked as needed. A much needed step in the right direction.

**Comment by oleg-nenashev** [comment]
It will collide with https://github.com/jenkins-infra/jenkins.io/pull/739

**Comment by oleg-nenashev** [comment]
Merging to see how it flies

**Pull request merged**